src/containers/Main.tsx

```diff
@@ -74,42 +89,111 @@ const AppAuthController = () => {
     }
   }, [wallets]);
 
+  /**
+   * If all the following variables are truthy within the if
+   * statement, we can consider this user as logged in and
+   * authenticated.
+   */
   if (isAppReady && authenticated && provider && chainId) {
-    return (
-        <EtherspotTransactionKit
-          provider={provider}
-          chainId={chainId}
-          bundlerApiKey={process.env.REACT_APP_ETHERSPOT_BUNDLER_API_KEY || undefined}
-          dataApiKey={process.env.REACT_APP_ETHERSPOT_DATA_API_KEY || undefined}
-        >
-          <AccountTransactionHistoryProvider>
-            <AssetsProvider>
-              <AccountBalancesProvider>
-                <AccountNftsProvider>
-                  <GlobalTransactionBatchesProvider>
-                    <BottomMenuModalProvider>
-                      <AuthContentWrapper>
-                        <AuthorizedNavigation />
-                      </AuthContentWrapper>
-                      <BottomMenu />
-                    </BottomMenuModalProvider>
-                  </GlobalTransactionBatchesProvider>
-                </AccountNftsProvider>
-              </AccountBalancesProvider>
-            </AssetsProvider>
-          </AccountTransactionHistoryProvider>
-        </EtherspotTransactionKit>
-    )
+
+    /**
+     * Define our authorized routes for users that are
+     * authenticated. There are a few steps here.
+     */
+
+    // First, add the core routes to the route definition
+    const authorizedRoutesDefinition = [{
+      path: '/',
+      element: <Authorized chainId={chainId} provider={provider}  />,
+      children: [
+        {
+          index: true,
+          path: '/',
+          element: <Lobby />,
+        },
+        {
+        path: '/landing',
+        element: <LandingPage />,
+      },
+      {
+        path: '/waitlist',
+        element: <Waitlist />,
+      }]
+    }];
+
+    // Next, add the allowed apps to the route definition
+    allowedApps.forEach((appId) => {
+      authorizedRoutesDefinition[0].children.push({
```

Comment:
Not sure to understand here why we have twice `authorizedRoutesDefinition[0].children.push()` ?

Reply:
Here is the full block:

```
   allowedApps.forEach((appId) => {
      authorizedRoutesDefinition[0].children.push({
        path: `/${appId}`,
        element: <App id={appId} />,
      });
      authorizedRoutesDefinition[0].children.push({
        path: `/${appId}/*`, // <-- is this what you are referring to? there's a catch all route here
        element: <App id={appId} />,
      });
    });
```

Reply:
Aaaah I see, ok thank you